src/routers/emailRouter.ts

```diff
@@ -40,20 +43,24 @@ router.post(config.CHANGE_EMAIL_ADDRESS_URL, async (req: Request, res: Response,
 });
 
 // GET: /check-your-answers
-router.get(config.CHECK_ANSWER_URL, async (req: Request, res: Response, next: NextFunction) => {
-  const handler = new ConfirmChangeEmailAddressHandler(
-    req.session?.data.signin_info?.user_profile?.email
-  );
-  const viewData = await handler.get(req, res);
-  res.render(`${routeViews}` + config.CHECK_ANSWER_URL, viewData);
+router.get(CHECK_ANSWER_URL, async (req: Request, res: Response, next: NextFunction) => {
+    await new CheckAnswerHandler().get(req, res)
+        .then((viewData) => {
+            res.render(`${routeViews}` + CHECK_ANSWER_URL, viewData);
+        });
 });
 
-// GET: /update-submitted
-router.get(config.UPDATE_SUBMITTED, async (req: Request, res: Response, next: NextFunction) => {
-  const handler = new UpdateSubmittedHandler();
-  await handler.get(req, res).then((viewData) => {
-    res.render(`${routeViews}` + config.UPDATE_SUBMITTED, viewData);
-  });
+// POST: /check-your-answers
+router.post(CHECK_ANSWER_URL, async (req: Request, res: Response, next: NextFunction) => {
+    await new CheckAnswerHandler().post(req, res)
+        .then((viewData) => {
+            if (Object.prototype.hasOwnProperty.call(viewData, errorsConst) ||
+                Object.prototype.hasOwnProperty.call(viewData, statementErrorsConst)) {
+                res.render(`${routeViews}` + CHECK_ANSWER_URL, viewData);
+            } else {
+                res.render(`${routeViews}` + SUBMITTED_URL, viewData);
+            }
+        });
```

Comment:
I've added the following route in email router but it seems to be missing here (maybe as a result of a conflict resolution)?

```suggestion
// GET: /update-submitted
router.get(config.UPDATE_SUBMITTED, async (req: Request, res: Response, next: NextFunction) => {
  const handler = new UpdateSubmittedHandler();
  await handler.get(req, res).then((viewData) => {
    res.render(`${routeViews}` + config.UPDATE_SUBMITTED, viewData);
  });
});
```

Reply:
done